README.md

```diff
@@ -26,6 +26,7 @@ Shipwright supports any tool that can build container images in Kubernetes clust
 
 - [Kaniko](https://github.com/GoogleContainerTools/kaniko)
 - [Cloud Native Buildpacks](https://buildpacks.io/)
+- [Buildkit](https://github.com/moby/buildkit)
```

Comment:
```suggestion
- [BuildKit](https://github.com/moby/buildkit)
```

Reply:
why the camel case?, name in upstream is [buildkit](https://github.com/moby/buildkit) without upper case.

Reply:
same applies to all parts where this change was suggested.

Reply:
See their header: https://github.com/moby/buildkit#buildkit

Reply:
thanks, also docker docs use BuildKit, `BuildKit` must be